cli/src/main.c

```diff
@@ -87,7 +85,7 @@ static int cmd_signature_message(const struct shell *shell, size_t argc, char **
         int offset = parse(argv[1], 96);
 
         if(offset != -1){
-            if(public_key_to_sk(argv[1], public_keys_hex_store, keys_counter, offset) != -1){
+            if(public_key_to_sk(argv[1], offset) != -1){
```

Comment:
The function public_key_to_sk is misleading. We better implement a function that returns true if the public key is in the keystore and false otherwise, e.g. pk_in_keystore. We should also remove the comment in line 94, it makes no sense now that the private key is in the secure world.

Reply:
Done